rero_ils/modules/items/api/record.py

```diff
@@ -24,25 +24,48 @@
 from ...libraries.api import Library
 from ...locations.api import Location
 from ...organisations.api import Organisation
-from ...utils import generate_item_barcode, trim_barcode_for_record
+from ...utils import extracted_data_from_ref, generate_item_barcode, \
+    get_ref_for_pid, trim_barcode_for_record
 
 
 class ItemRecord(IlsRecord):
     """Item record class."""
 
+    def extended_validation(self, **kwargs):
+        """Add additional record validation.
+
+        Ensures that item of type issue is created only on
+        holdings of type serials.
+
+        Ensures that item of type issue has the issue field.
+
+        Ensures that standard item has no issue field.
+
+        :returns: False if
```

Comment:
:returns: -> :return:
can you replace similar mistake in the code?

Reply:
fixed